kfs-core/src/main/resources/edu/arizona/kfs/coa/businessobject/datadictionary/OrgReviewRole.xml

```diff
@@ -41,7 +41,20 @@
       </list>
 	</property>    
   </bean>
-		
+
+  <bean id="OrgReviewRole-reviewRolesIndicator" parent="OrgReviewRole-reviewRolesIndicator-parentBean" />
+  <bean id="OrgReviewRole-reviewRolesIndicator-parentBean" abstract="true" parent="AttributeDefinition">
```

Comment:
I think this was just a case where I copied the entire attribute definition and made the change to the control. I can make the change to only override the control and retest.